### PR TITLE
chore: make the code more concise, and fix some minor issues

### PR DESCRIPTION
**Pull request by dirname**

### CODE REVIEW

File: casdoc_test.go [hunk 1]
```diff
@@ -15,78 +15,53 @@
...
-	log "github.com/sirupsen/logrus"
 )
 
 func TestPolishDocs(t *testing.T) {
 	Q := utils.WorkQueue{}
 	err := Q.GetFileList(path.Join(config.RepoPath, "/docs/"))
 	if err != nil {
-		Logger.Errorf("Failed to get file list")
-		panic(err)
+		t.Fatalf("Failed to get file list, err: %v", err)
```
Comment (dirname) [inline casdoc_test.go:31]: In unit testing, I believe using the `t.Fatalf` to terminate the test is more appropriate for the scenario.

File: casdoc_test.go [hunk 2]
```diff
@@ -15,78 +15,53 @@
...
 	}
 
-	counter := 0
 	totalItems := len(Q.Item)
 
-	for {
+	for counter := 1; !Q.Empty(); counter++ {
```
Comment (dirname) [inline casdoc_test.go:36]: Make the code more concise.

File: prompt/prompt.go [hunk 3]
```diff
@@ -16,94 +16,47 @@ package prompt
...
-
-var JapaneseRequest = openai.ChatCompletionRequest{
-	Model: openai.GPT3Dot5Turbo,
-	Messages: []openai.ChatCompletionMessage{
-		{
-			Role:    openai.ChatMessageRoleSystem,
-			Content: japanesePrompt,
-		},
-	},
+var LanguageRequests = map[string]openai.ChatCompletionRequest{
```
Comment (dirname) [inline prompt/prompt.go:54]: Use map and factory functions to avoid redundant code and make the code more concise.

File: utils/workQueue.go [hunk 4]
```diff
@@ -27,10 +27,13 @@ type WorkQueue struct {
...
 func (q *WorkQueue) GetFileList(docDir string) error {
+	isMarkdownFile := func(path string) bool {
+		ext := filepath.Ext(path)
+		return ext == ".md" || ext == ".mdx"
+	}
+
 	err := filepath.Walk(docDir, func(path string, info fs.FileInfo, err error) error {
-		var name = info.Name()
-		var size = len(name)
-		if !info.IsDir() && (name[size-3:] == "mdx" || name[size-2:] == "md") {
```
Comment (dirname) [inline utils/workQueue.go:33]: Use the `path/filepath` package to determine the file extension.

**Review by dirname — COMMENTED** [review]
#1

File: config/config.go [hunk 5]
```diff
@@ -47,27 +49,23 @@ func init() {
...
-	if AuthToken == "" {
-		fmt.Println("Not found AuthToken, please type it: ")
-		var input string
-		_, _ = fmt.Scanln(&input)
-		AuthToken = input
-		if AuthToken == "" {
-			panic("AuthToken is empty")
-		}
-	}
+	checkEmptyValue(&AuthToken, "AuthToken")
```
Comment (dirname) [inline config/config.go:52]: Use functions to avoid duplicating the same code.

File: config/config.go [hunk 6]
```diff
@@ -28,16 +29,17 @@ var (
 )
 
 func init() {
-	viper.SetDefault("AuthToken", "")
```
Comment (dirname) [inline config/config.go:31]: This is redundant.

**Comment by hsluoyz** [comment]
@dirname casdoc will finally be merged into: https://github.com/casbin/casibase , this repo will be deprecated.

**Pull request closed without merging**